man/legend.grob.Rd

```diff
@@ -45,7 +45,7 @@ The 'colours' component is a vector of fill colours to be used for the rectangle
 }
 \value{Returns an grob representing the legend(s)}
 \author{Lauren Chong}
-\seealso{\code{\link{create.heatmap}}, \code{\link{draw.key}}, \code{\link{gpar}}}
+\seealso{\code{\link{create.heatmap}}, \code{\link[lattice]{draw.key}}, \code{\link{gpar}}}
```

Comment:
does this `\code{\link{gpar}}` have to be `\code{\link[grid]{gpar}}`? ordo we not need to do that for base R stuff?

Reply:
> does this `\code{\link{gpar}}` have to be `\code{\link[grid]{gpar}}`? ordo we not need to do that for base R stuff?

I thought about that! I think that's the case, or something along those lines. R CMD check only had a problem with lattice - not grid. It may also be that `gpar` is used in the NAMESPACE, so it can automatically resolve.